src/chainparams.cpp

```diff
@@ -298,3 +320,24 @@ bool SelectParamsFromCommandLine()
     SelectParams(network);
     return true;
 }
+
+
+// Block height must be >0 and less than the halving interval block height
+// Index variable i ranges from 0 - (vFoundersRewardAddress.size()-1)
+std::string CChainParams::GetFoundersRewardAddress(size_t nHeight) const
+{
+    assert(nHeight>0 && nHeight<consensus.nSubsidyHalvingInterval);
+    size_t i = (size_t)floor((double(nHeight)/consensus.nSubsidyHalvingInterval)*vFoundersRewardAddress.size());
```

Comment:
@daira
> It seems like there is some inconsistency in the duration of the Founders' Reward. I had been under the impression that the Founders' Reward lasted exactly until the first halvening.

This is supposed to be the case; the mining slow start was calibrated such that the total monetary base at first halving was unchanged by it. If the Founders' Reward is not consistent with this, that is a critical consensus bug that we need to fix in beta 2.